### PR TITLE
  avg of 10 numbers(array)

### PR DESCRIPTION
**Pull request by anchal071**
coding for finding avg of 10 numbers using array.

### CODE REVIEW

File: avg of 10 numbers(array) [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+#include<conio.h>
+int main()
+{
+  int a[10],i,sum = 0;
+  float avg;
+  printf("enter 10 numbers");
+  for(i= 0;i< = 9;i++)
```
Comment (Anjali-1801) [inline avg of 10 numbers(array):8]: There shouldn't be any space while using a conditional operator. Replace i< = 9 with i<=9.

Comment (sarvesh-0) [reply avg of 10 numbers(array):8]: @anchal071 Please make the changes suggested by @Anjali-1801.

**Comment by anchal071** [comment]
Ok sir.

File: avg of 10 numbers(array) (hunk 1 above)
Comment (anchal071) [reply avg of 10 numbers(array):8]: sir ; I  also want to make changes suggested by her but I have no idea that how to do that.

Comment (Anjali-1801) [reply avg of 10 numbers(array):8]: Make the changes in your cloned repository (in local device) and then push the changes, it'll reflect in your PR.

**Comment by anchal071** [comment]
Ok. I will do. Thank you so much.....

**Review by anchal071 — COMMENTED** [review]
i< =9 changes to i<=9

**Pull request merged**